.github/workflows/run_tests.yml

```diff
@@ -2,38 +2,25 @@ name: test
 
 on:
   push:
+    branches:
+      - main
+      - master
     paths:
       - '**.py'
       - '.github/workflows/test*.yml'
       - 'pyproject.toml'
+      - 'poetry.lock'
+      - 'requirements/requirements*.txt'
   pull_request:
     paths:
       - '**.py'
       - '.github/workflows/test*.yml'
       - 'pyproject.toml'
+      - 'poetry.lock'
+      - 'requirements/requirements*.txt'
 
 jobs:
-  style:
-    name: check code style
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@main
-      - uses: actions/setup-python@main
-        with:
-          python-version: '3.x'
-      - uses: actions/cache@main
-        id: cache
-        with:
-          path: ${{ env.pythonLocation }}
-          key: style-${{ runner.os }}-${{ env.pythonLocation }}-${{ hashFiles('pyproject.toml', 'setup.*', 'requirements/*') }}
-      - run: pip install flake8 black
-      # stop the build if there are Python syntax errors or undefined names
-      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-      - run: black . --check
```

Comment:
Isn't it going to be harder to enforce style if we don't include them in the tests? If we have this at least even if the dev env on the local machine fails to capture, we know the style issue is captured here, right?